Nos/Assets/Localization/Localizable.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import SwiftUI
+
+/// A protocol for strings that are translated into multiple languages. See `Localized` for concrete implementation and
+/// usage docs.
+protocol Localizable: RawRepresentable<String> {
+    
+    var template: String { get }
+
+    /// optionally override this to provide your own key
+    var key: String { get }
+
+    /// optionally override this to provide your own namespace key
+    /// defaults to the type name of the enum
+    static var namespace: String { get }
+    
+    /// Creates a SwiftUI Text view for this text.
+    var view: Text { get }
+
+    static func exportForStringsFile() -> String
+}
+
+extension Localizable {
+    
+    // You can modify this to perform localization, or overrides based on server or other config
+    var string: String {
+        rawValue
+//        let bundle = Bundle.current
+//        return NSLocalizedString(key, tableName: "Generated", bundle: bundle, comment: "")
```

Comment:
This'll be used when the Generated.strings is added, right? I think it'd be nice to have a TODO comment here to explain the commented code, but it's not that big of a deal

Reply:
Good idea, fixed!